decidim-admin/app/forms/decidim/admin/scope_form.rb

```diff
@@ -4,21 +4,29 @@ module Decidim
   module Admin
     # A form object to create or update scopes.
     class ScopeForm < Form
-      attribute :name, String
+      include TranslatableAttributes
+
+      translatable_attribute :name, String
       attribute :organization, Decidim::Organization
+      attribute :code, String
+      attribute :parent_id, Integer
+      attribute :scope_type_id, Integer
+      attribute :deprecated, Boolean
```

Comment:
Yes, old processes should keep it original scope. Also, users related to that scope may be asked to choose another scope.
Thinking about it, I think my solution is incomplete. It marks deprecated territories on participatory process edition form, but it should also hide those territories in new processes, while keeping it on old ones. 
Maybe the best solution would be to change deprecated column to a deprecated_at datetime. That way, scope elegibility and searchability would depend on participatory process creation date. What do you think?